package.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "core-js": "^3.9.1",
-    "firebase": "^8.3.1",
+    "firebase": "^8.2.9",
```

Comment:
Is reverting to this older version really necessary?

Reply:
with 8.3.1 I could not successfully build the project, due to an error in node_modules/firebase/... . After cleaning up and reinstalling node_modules several times, I gave up and reverted to previous version.

Reply:
> Is reverting to this older version really necessary?

with 8.3.1 I could not successfully build the project, due to an error in node_modules/firebase/... . After cleaning up and reinstalling node_modules several times, I gave up and reverted to previous version.